app/frontend/controllers/app_controller.js

```diff
@@ -140,6 +141,50 @@ class AppController {
                              defaultBibleTranslationId);
   }
 
+  initVerseButtons(tabIndex=undefined) {
+    var currentVerseListMenu = app_controller.getCurrentVerseListMenu(tabIndex);
+    var editNoteButton = currentVerseListMenu.find('.edit-note-button');
+    var copyButton = currentVerseListMenu.find('.copy-clipboard-button');
+
+    editNoteButton.bind('click', (event) => {
+      event.stopPropagation();
+
+      if (!$(this).hasClass('ui-state-disabled')) {
+        app_controller.hideAllMenus();
+        app_controller.notes_controller.editVerseNotesForCurrentlySelectedVerse();
+      }
+    });
+
+    copyButton.bind('click', (event) => {
+      event.stopPropagation();
+
+      if (!$(this).hasClass('ui-state-disabled')) {
+        app_controller.hideAllMenus();
+        app_controller.copySelectedVerseTextToClipboard();
+      }
+    });
+
+    this.assign_last_tag_button.init(tabIndex);
+    this.translationComparison.initButtonEvents();
+    this.verse_context_controller.initButtonEvents();
+  }
+
+  enableVerseButtons() {
+    var currentVerseListMenu = app_controller.getCurrentVerseListMenu();
+    var editNoteButton = currentVerseListMenu.find('.edit-note-button');
+    var copyButton = currentVerseListMenu.find('.copy-clipboard-button');
+    editNoteButton[0].classList.remove('ui-state-disabled');
+    copyButton[0].classList.remove('ui-state-disabled');
```

Comment:
This can be refactored in vanilla javascript without jQuery:
```javascript
    var currentVerseListMenu = app_controller.getCurrentVerseListMenu()[0];
    var editNoteButton = currentVerseListMenu.querySelector('.edit-note-button');
    var copyButton = currentVerseListMenu.querySelector('.copy-clipboard-button');
    editNoteButton.classList.remove('ui-state-disabled');
    copyButton.classList.remove('ui-state-disabled');
```

Reply:
Ok